Podverse/MediaRef.swift

```diff
@@ -138,4 +145,79 @@ class MediaRef {
             return "(untitled clip)"
         }
     }
+    
+    // :(
+    static func convertPlayerHistoryItemToMediaRefPostString(item: PlayerHistoryItem, shouldSaveFullEpisode: Bool = false) -> String {
```

Comment:
Do you think this method should instead be a PlayerHistoryItem method instead? That way you could call playerHistoryItem.toMediaRefPostString(shouldSaveFullEpisode)